modules/aws/ebs/main.tf

```diff
@@ -0,0 +1,75 @@
+/**
+ * AWS EBS
+ * =======
+ * This module creates additional ebs volumes and attaches them to the instances.
+ *
+ * EXAMPLE
+ * -------
+ *
+ *```hcl
+ * module "instance-volume" {
+ *   source = "dcos-terraform/ebs/aws"
+ *
+ *   instances          = ["id-ablsldhfa123", "id-aidrkdkek131"]
+ *   num                = "3"
+ *   sizes              = ["500", "100", "1000"]
+ *   types              = ["", "", "gp2"]
+ *   iopses             = ["0", "300", "3000"]
```

Comment:
My first impression is that three independent arrays are not good idea. I would prefer a list of maps. Any reason why you have chosen such a design?

Reply:
yes. an example are additional_listeners here https://github.com/dcos-terraform/terraform-aws-lb

However this also requires handling unset keys and defaults. But in general this is possible

Reply:
ah! very nice! i like that! I'll see what I can do.